OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -85,7 +88,11 @@ public MainMenuLogic(Widget widget, World world, ModData modData)
 
 			mainMenu.Get<ButtonWidget>("EXTRAS_BUTTON").OnClick = () => SwitchMenu(MenuType.Extras);
 
-			mainMenu.Get<ButtonWidget>("QUIT_BUTTON").OnClick = Game.Exit;
+			mainMenu.Get<ButtonWidget>("QUIT_BUTTON").OnClick = () =>
```

Comment:
There are a bunch of ways we could be exiting the game without clicking this button (Window buttons, alt-f4, crash, switching mods when joining a server). We could instead make `IGlobalModData : IDisposable` and wire up `ModData.Dispose` &rarr; `Manifest.Dispose` &rarr; `IGlobalModData.Dispose` to do this automatically.